api/swagger.yaml

```diff
@@ -1988,6 +1988,19 @@ definitions:
         x-nullable: false
         type: "integer"
         example: 2
+      PlatformImages:
+        description: |
+          Platform-specific images available for this image.
+
+          Only present with the containerd integration enabled.
+
+          WARNING: This is experimental and may change at any time without any backward
+          compatibility.
+        type: "array"
+        x-nullable: false
+        x-omitempty: true
```

Comment:
Hmm CI fails with:
```
diff --git a/api/types/image/summary.go b/api/types/image/summary.go
index 4ebbdeadf7..fed2746521 100644
--- a/api/types/image/summary.go
+++ b/api/types/image/summary.go
@@ -54,7 +54,7 @@ type Summary struct {
 	// WARNING: This is experimental and may change at any time without any backward
 	// compatibility.
 	//
-	PlatformImages []PlatformImage `json:"PlatformImages,omitempty"`
+	PlatformImages []PlatformImage `json:"PlatformImages"`
 
 	// List of content-addressable digests of locally available image manifests
 	// that the image is referenced from. Multiple manifests can refer to the

Please update api/swagger.yaml with any API changes, then 
run hack/generate-swagger-api.sh.
```

Even though I marked this one with `x-omitempty` 🤔

Reply:
Looks like it's an issue with the outdated go-swagger, see https://github.com/moby/moby/pull/47827 for update